internal/core/api.rs

```diff
@@ -594,7 +594,7 @@ impl Window {
     pub fn try_dispatch_event(
         &self,
         event: crate::platform::WindowEvent,
-    ) -> Result<(), PlatformError> {
+    ) -> Result<bool, PlatformError> {
```

Comment:
This is a public API and changing that is a breaking change.

It should be possible to use the private API `process_key_input` directly from the android platform.

Reply:
(btw, this one is already marked as one of the breaking change for 2.0 https://github.com/slint-ui/slint/issues/3023 )